Resources/views/Collector/mongodb.html.twig

```diff
@@ -68,7 +68,9 @@
                 <td class="nowrap">{{ loop.index }}</td>
                 <td class="nowrap">{{ '%0.3f'|format(command.durationMicros / 1000) }} ms</td>
                 <td class="nowrap">{{ command.database }}</td>
-                <td>{{ command.command }}</td>
+                <td>
+                    <pre>{{ command.command|json_encode(constant('JSON_PRETTY_PRINT')) }}</pre>
```

Comment:
I think we should show the non-pretty-print one first and allow to expand it for the pretty print one. You can use `data-toggle-selector` attributes to achieve it. See https://github.com/doctrine/DoctrineBundle/blob/2.4.x/Resources/views/Collector/db.html.twig

Reply:
Done by using existing `queries-table` and `sql-runnable` CSS classes.